rosidl_tutorials/rosidl_tutorials/msg/AddressBook.msg

```diff
@@ -0,0 +1 @@
+rosidl_tutorials_msgs/Contact[] address_book
```

Comment:
I would suggest to call the field `contacts`. The whole message represents an address book. The field itself only a list of contacts.